pulse2percept/implants/prima.py

```diff
@@ -0,0 +1,28 @@
+"""`Prima`"""
+import numpy as np
+from collections import OrderedDict
+from pulse2percept.implants import ElectrodeGrid, ProsthesisSystem, DiskElectrode
+
+class Prima(ProsthesisSystem):
```

Comment:
This is fine, but don't forget to also add it to `__init__.py` so it can be imported